tests/software_tests/can/addressing/test_normal_addressing.py

```
@@ -0,0 +1,569 @@
+        self.mock_addressing_information.rx_functional_params = rx_functional_params
+        self.mock_addressing_information.tx_functional_params = tx_functional_params
+        assert NormalCanAddressingInformation._validate_addressing_information(self.mock_addressing_information) is None
+
+    # validate_addressing_params
+
+    @pytest.mark.parametrize("addressing_format", [
+        Mock(),
+        CanAddressingFormat.EXTENDED_ADDRESSING,
+    ])
+    def test_validate_addressing_params__value_error(self, addressing_format):
+        with pytest.raises(ValueError):
```

Ruff (0.12.2)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)